### DISCUSSION TITLE
Purge Web Analytics data

### DISCUSSION THREAD

**Discussion by stitot** [vercel/next.js#53641] [Ideas]
### Goals

1. To be able to purge Web Analytics data

### Background

When implementing events it's quite often necessary to refine/rename/consolidate/tweak/whatever the wording and/or number of events and corresponding custom data, even after things goes into production. It would be nice to be able to purge data, just to do a cleanup and avoid pollution.

### Proposal

E.g. a listing of all tracked events at https://vercel.com/username/project/analytics with checkboxes. Check x number of events and hit a delete/purge/whatever button.

**Comment by craigRSA** [comment]
I agree, we moved a page to production after testing for a few months and so like clean Analytics for it

**Comment by XcovertX** [comment]
Does this mean there is currently no way to purge the events log?

**Comment by felixmpaulus** [comment]
Interested as well...
